### PR TITLE
Sort Proposals

### PR DESCRIPTION
**Pull request by roienatan**
resolves: https://github.com/daostack/alchemy/issues/2326

Sort proposals by the next order: executables, passing and falling by `closingAt` in ascending order, and then failed and executed proposals.

### CODE REVIEW

**Comment by orenyodfat** [comment]
filter executed proposal here https://github.com/daostack/alchemy/blob/dev/src/components/Redemptions/RedemptionsPage.tsx#L250

File: test/integration/proposal-genericSchemeEnsRegistry.e2e.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ describe("Proposals ENS Registry", () => {
 
   it("Create a Generic Scheme ENS Registry proposal and check that the data is submitted correctly", async () => {
     await gotoDaoSchemes(daoAddress);
+    await hideCookieAcceptWindow();
```
Comment (orenyodfat) [inline test/integration/proposal-genericSchemeEnsRegistry.e2e.ts:19]: why is this needed ?

Comment (roienatan) [reply test/integration/proposal-genericSchemeEnsRegistry.e2e.ts:19]: For some reason this test failed on Travis and locally because the cookies window blocked it. Anyway it's harmless it just hides the cookies prompt.

**Comment by roienatan** [comment]
> filter executed proposal here https://github.com/daostack/alchemy/blob/dev/src/components/Redemptions/RedemptionsPage.tsx#L250

I think we should leave it as it is, with this change the redeem test fails because it has nothing to redeem. It already fetches the proposals with this filter `accountsWithUnclaimedRewards_contains`. I also verified it on a proposal on Rinkeby, with this filter I have nothing to redeem but without it I can see what I can redeem. Let me know what you think.

**Review by Razzwan — APPROVED** [review]

File: test/integration/proposal-genericSchemeEnsRegistry.e2e.ts (hunk 1 above)
Comment (Razzwan) [reply test/integration/proposal-genericSchemeEnsRegistry.e2e.ts:19]: @roienatan it can be a problem for other tests as well? Can you show the video, how does it looks like?

**Review by Razzwan — APPROVED** [review]

File: test/integration/proposal-genericSchemeEnsRegistry.e2e.ts (hunk 1 above)
Comment (roienatan) [reply test/integration/proposal-genericSchemeEnsRegistry.e2e.ts:19]: As I saw it's not consistent. Will explain you when we talk.
